app/Models/Profile.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Profile extends Model
+{
+    use HasFactory;
+
+    protected $table = "profile";
+    protected $guarded  = "";
+
+    public function scopeCurrent($query, $userId)
+    {
+        return $query->where('user_id', $userId)->first();
```

Comment:
Ko dùng scope ở đây, mà hãy định nghĩa relation trong model.
Dùng các mối quan hệ has one, belong to là đc.